### ISSUE TITLE
build_files managment command is broken

### ISSUE THREAD

**Issue by gregmuellegger** [rtfd/readthedocs.org#1512]
The management command cannot work since it's calling `project.tasks.fileify` without the `commit` argument: 

The `commit` argument was added in Oct 2014 to the function: https://github.com/rtfd/readthedocs.org/blob/291d7d61e626c1dfb998d9e5d4e5b734a44a7024/readthedocs/projects/tasks.py#L704

That means the management command is broken since then. Is the command still in active use? We can either fix it somehow or remove it completely.

btw, this was uncovered by prospector/pylint. Linting rules :candy: 
```
management/commands/build_files.py
  Line: 46
    pylint: no-value-for-parameter / No value for argument 'commit' in function call (col 16)
```

**Comment by ericholscher** [comment]
We don't actively use it, but it was created to help populate things if your data gets out of sync. We can probably safely remove it for now, but we should implement a way to do a full "finish build" on all projects to enable us to reindex everything that happens after a build, in case we get out of sync in the future.

**Comment by gregmuellegger** [comment]
Will be fixed by #1525
